R/run.R

```diff
@@ -35,12 +36,20 @@
 #' @importFrom withr with_output_sink
 #' @export
 run <- function(renvInstallPackages = NULL,
-                computations = baseComputations[c("magpiePrep", "remindPrep")],
+                computations = c("magpiePrep", "remindPrep"),
                 piktestsFolder = getwd(),
                 runFolder = NULL,
                 jobNameSuffix = "",
                 executionMode = c("slurm", "directly"),
                 localCache = TRUE) {
+
+  if (!is.list(computations)) {
```

Comment:
```suggestion
  if (is.character(computations)) {
```
Seems clearer to me, also we can assume we have a character vector now.

Reply:
with the approach I proposed we will also catch other misconfigurations which we other wise would not catch. Therefore, I suggest to stick to my formulation